js/pages/backoffice/toolbox.js

```diff
@@ -323,6 +348,85 @@ CombodoModal._CenterModalInViewport = function (oModalElem) {
 	});
 };
 
+/**
+ * Open a standard confirmation modal and put the content into it.
+ *
+ * @param oOptions array @see CombodoModal.OpenModal + {do_not_show_again_pref_key: string, callback_on_confirm: function, callback_on_cancel}
+ * @param aData data passed to callbacks
+ * @returns object The jQuery object of the modal element
+ */
+CombodoModal.OpenConfirmationModal = function(oOptions, aData) {
+
+	// check do not show again preference key
+	if(oOptions.do_not_show_again_pref_key != null){
+		if(GetUserPreference(oOptions.do_not_show_again_pref_key, false)){
+			if(oOptions.callback_on_confirm != null){
```

Comment:
```suggestion
	// Check do not show again preference key
	if(oOptions.do_not_show_again_pref_key !== null){
		if(GetUserPreference(oOptions.do_not_show_again_pref_key, false)){
			if(oOptions.callback_on_confirm !== null){
```

We tend not to rely on type conversion